services/klatchat_observer/controller.py

```diff
@@ -847,6 +855,28 @@ def on_get_configured_personas(self, body: dict):
             expiration=5000,
         )
 
+    @create_mq_callback()
+    def handle_prompts_data_request(self, body: dict):
```

Comment:
Please add a docstring and consider refactoring the method name to be more descriptive, i.e.
```suggestion
    def handle_get_prompts_data(self, body: dict):
    """
    Handle a client request to get configured Chatbotsforum prompts
    """
```